examples/DotNetCore/WebJobs/WebJobHelloWorld/WebJobHelloWorld/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Configuration;
+
+namespace WebJobHelloWorld
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var hostBuilder = new HostBuilder();
+            hostBuilder.ConfigureHostConfiguration((config) =>
+            {
+                LoadConfiguration(config);
+            });
+            hostBuilder.ConfigureWebJobs(b =>
+            {
+                b.AddAzureStorageCoreServices();
+                b.AddAzureStorage();
+            });
+            hostBuilder.ConfigureLogging((context, b) =>
+            {
+                b.AddConsole();
+            });
+
+            var host = hostBuilder.Build();
+            using (host)
+            {
+                host.Run();
+            }
+        }
+
+        private static void LoadConfiguration(IConfigurationBuilder configBuilder)
+        {
+            //
+            // This application attempts to connect to Azure App Configuration to retrieve Azure Blob Storage name, and Queue Name for the Azure Web Job.
+            // It reads the ConnectionString for the App Configuration Service from appsettings.json and from environment variables.
```

Comment:
>appsettings.json [](start = 84, length = 16)

I think we should drop appsettings.json here. Users should never put connection string in appsettings.json.

Reply:
Removed appSettings.json from this example